.golangci.next.reference.yml

```diff
@@ -137,6 +137,7 @@ linters:
     - bidichk
     - bodyclose
     - canonicalheader
+    - clearslice
```

Comment:
Disabled by default to minimize adoption annoyance (and anticipating some iteration/improvement over time)